ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml

```diff
@@ -152,7 +152,6 @@ GenerateOfflineMapLocalBasemapSample {
     /* Uncomment this section when running as standalone application
     AuthenticationView {
         anchors.fill: parent
-        authenticationManager: offlineMapSample.authenticationManager // set the authenticationManager property (this needs to be registered)
```

Comment:
@anmacdonald @JamesMBallard - Just a thought - removing this property is great and simplifies the code. It also seems to be the most common recurring code change in the PR - everything else is related to imports and registration. Should we consider adding a dummy no-op property on AuthenticationView called authenticationManager just for portability? Everything will work whether you set it or not, but it won't require code change.

My initial thought is no, because the QML Engine will just fail to load the QML file if you try to set a non existent property, so it should be pretty obvious and easy to change this.

Reply:
There are more AuthenticationView declarations in this PR than I think we'd see in a typical application, so I'd say the benefit wouldn't be as effective in the wild.

Reply:
I would prefer to avoid no-op properties. It would ease porting apps to the new toolkit, but we should wait to see if we hear feedback from users and consider adding it later.

Reply:
yep I agree. Just wanted to throw it out there, as Qt is doing something similar with QList & QVector so as to make porting easier to Qt 6 https://www.qt.io/blog/qlist-changes-in-qt-6. In this case, this code is going to only be set once in an app, so it shouldn't be much to change